bundle/src/main/java/com/adobe/acs/commons/adobeio/core/service/HttpClientService.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * #%L
+ * ACS AEM Commons Bundle
+ * %%
+ * Copyright (C) 2013 - 2018 Adobe
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+package com.adobe.acs.commons.adobeio.core.service;
+
+import org.apache.http.impl.client.CloseableHttpClient;
+
+public interface HttpClientService {
```

Comment:
Is this a general-purpose service or something intended only to be used by this integration? If the former, don't we already have this? If the latter, why is a service necessary (and it shouldn't be API).